classes/sota_raspberrypi.bbclass

```diff
@@ -1,11 +1,22 @@
 RPI_USE_U_BOOT_sota = "1"
-KERNEL_IMAGETYPE_sota = "uImage"
+
+KERNEL_CLASSES_append_sota = " kernel-fitimage"
+KERNEL_IMAGETYPE_sota = "fitImage"
+
 PREFERRED_PROVIDER_virtual/bootloader_sota ?= "u-boot"
 UBOOT_MACHINE_raspberrypi2_sota ?= "rpi_2_defconfig"
-UBOOT_MACHINE_raspberrypi3_sota ?= "rpi_3_32b_defconfig"
+UBOOT_MACHIE_raspberrypi3_sota ?= "rpi_3_32b_defconfig"
```

Comment:
Typo? Should be `UBOOT_MACHINE...`, right?

Reply:
Hm... I wonder how it worked for me.

Reply:
Actually these assignments are redundant, it's already done in machine configuration.